pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -397,9 +397,18 @@ func newPrefixWriter(out io.Writer, prefix string) io.Writer {
 	return writer
 }
 
+func (o *MustGatherOptions) getBackoff() wait.Backoff {
+	return wait.Backoff{
+		Duration: time.Second,
+		Factor:   2,
+		Steps:    10,
+		Cap:      time.Duration(o.Timeout) * time.Second,
+	}
+}
+
 func (o *MustGatherOptions) waitForPodRunning(pod *corev1.Pod) error {
 	phase := pod.Status.Phase
-	err := wait.PollImmediate(time.Minute, time.Duration(o.Timeout)*time.Second, func() (bool, error) {
+	err := wait.ExponentialBackoff(o.getBackoff(), func() (bool, error) {
```

Comment:
Does shortening this initial timeout like here https://github.com/openshift/oc/pull/527 does the trick for you? Why you're proposing the exponential backoff?

Reply:
I thought you needed a longer interval, so I figure this was a decent compromise.  If #527 works for you, then yes, it will solve our issue as well and we can close this.